library/plone

```diff
@@ -0,0 +1,43 @@
+Maintainers: Alin Voinea alin.voinea@gmail.com (@avoinea),
+             Sven Strack sven@so36.net (@svx)
+GitRepo: https://github.com/plone/plone.docker.git
+
+Tags: 5.0.6, 5.0, 5, latest
+GitCommit: 443c34c35e774820621bf5c0966c81c3b45cb284
+Directory: 5.0/5.0.6/debian
+
+Tags: 5.0.5
+GitCommit: 443c34c35e774820621bf5c0966c81c3b45cb284
+Directory: 5.0/5.0.5/debian
```

Comment:
Are these old versions officially supported upstream, or are you simply looking to backfill the tags and looking to remove them after they're built?  (See also https://github.com/docker-library/official-images#tags-and-aliases)

To put it another way, if there's a problem discovered in 5.0.5, will there be a 5.0.5.1 release, or is that what 5.0.6 is?  Is there a compelling reason why a user would need to use 5.0.5 and couldn't upgrade to 5.0.6?

When tags are removed here, they do _not_ get removed from the Docker Hub, so are still available for users to pull as desired.